custom-schematics/src/token-option/files/custom-resolver/__name@dasherize__-token-option-resolver.ts

```diff
@@ -0,0 +1,19 @@
+import type { TokenOptionGroup } from 'app/data/token-option-group';
+import type { <%= classify(name) %>TokenOption } from './<%= dasherize(name) %>-token-option';
+import type { ExtractFromDataType } from 'app/token-options/mixins/from-data-mixin';
+import { TokenOptionResolver } from 'app/token-options/token-option-resolver';
+
+@Injectable()
```

Comment:
Thank you for catching that! This decorator is used to let Angular DI know that this class supports dependency injection. Without `@Injectable()` decorator, this class itself could still be injected into other services, but it cannot have any injected services (i.e., cannot use `@Inject()` in `constructor()`). Since the need of dependency injection is quite common, I added the `@Injectable()` annotation here. I will add the import for it in the next commit.